<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1269</width>
    <height>948</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа #6</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>951</width>
      <height>891</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="color_ind_edge">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QPushButton" name="but_color_edge">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выбрать
цвет
контура</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_x">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="but_color_bg">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выбрать
цвет
фона</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_color_edge_def">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>По
умочанию</string>
    </property>
   </widget>
   <widget class="QLabel" name="color_ind_bg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QPushButton" name="but_color_bg_def">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>По
умочанию</string>
    </property>
   </widget>
   <widget class="QLabel" name="color_ind_fill">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QPushButton" name="but_color_fill">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выбрать
цвет
закраски</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_color_fill_def">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>По
умочанию</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_x">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lab_y">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_y">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="but_add_dot">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить
точку</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_close">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Замкнуть фигуру</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_centre_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удерживайте SHIFT, чтобы
добавить вертикальную линию</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lab_centre_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удерживайте CTRL, чтобы
добавить горизонтальную линию</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="but_clear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>680</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Очистить экран</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Включить задержку</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_centre_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Изменения цвета контура и фона
вступают в силу после очистки экрана</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="rad_edge">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить точку границы</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rad_dot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить затравочный пиксель</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_close_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Нарисовать область шириной
в один пиксель</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1269</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
